### ISSUE TITLE
Current emacs.aliases.bash aliases 'et' twice overwriting one of the versions.

### ISSUE THREAD

**Issue by ntwyman** [Bash-it/bash-it#578]
Below is the current checkout. Notice that the 'emacsclient -t' definition overwrites the 'emacs -nw definition'. FWIW I have aliased en for the 'emacs -nw' version

```bash
± |master ✓| → cat ~/.bash_it/aliases/available/emacs.aliases.bash 
cite 'about-alias'
about-alias 'emacs editor'

case $OSTYPE in
  linux*)
    alias em='emacs'
    alias et='emacs -nw'
    alias e='emacsclient -n'
    alias et='emacsclient -t'
    alias ed='emacs --daemon'
    alias E='SUDO_EDITOR="emacsclient" sudo -e'
    ;;
  darwin*)
    alias em='open -a emacs'
    ;;
esac
```

**Comment by nwinkler** [comment]
Thanks for bringing this up! How about a Pull Request to fix this?

**Comment by ntwyman** [comment]
Will do.

**Comment by nwinkler** [comment]
Closed im #580
